sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_client_async.py

```diff
@@ -246,7 +245,7 @@ async def open_async(self, connection=None):
             self._connection = Connection(
                 "amqps://" + self._hostname if self._use_tls else "amqp://" + self._hostname,
                 sasl_credential=self._auth.sasl,
-                ssl_opts={'ca_certs': self._connection_verify or certifi.where()},
```

Comment:
is certifi.where no longer needed?

Reply:
I moved the certifi.where into the client constructor, which is inherited from the sync client.